pkg/container-collection/container-collection.go

```diff
@@ -51,6 +51,12 @@ type ContainerCollection struct {
 
 	// initialized tells if ContainerCollectionInitialize has been called.
 	initialized bool
+
+	// initialized tells if ContainerCollectionInitialize has been closed.
+	closed bool
```

Comment:
```suggestion
	// closed tells if ContainerCollectionInitialize has been closed.
	closed bool
```

Reply:
ops. Thanks for catching this up.

Reply:
I still see:

```golang
	// initialized tells if ContainerCollectionInitialize has been closed.
	closed bool
```

Reply:
You're right. I messed up with a force push and pushed the wrong version. It should be fixed now.